src/filters/renderer/VideoRenderers/DX9RenderingEngine.cpp

```diff
@@ -390,20 +390,28 @@ HRESULT CDX9RenderingEngine::RenderVideoDrawPath(IDirect3DSurface9* pRenderTarge
             start = stop;    // reset after 10 min (ps float has its limits in both range and accuracy)
         }
 
+        LONGLONG lCurrentPosition = 0;
+        LONGLONG lDuration = 0;
+        CComPtr<IMediaStatusPresenter> pInfos;
+        if (SUCCEEDED(GetMediaStatusPresenter(&pInfos))) {
+            pInfos->GetMediaTimes(&lCurrentPosition, &lDuration);
+        }
+        const float clTickToSeconds = 1.0f / 10000000.0f;
+
 #if 1
         D3DSURFACE_DESC desc;
         m_pVideoTexture[m_nCurSurface]->GetLevelDesc(0, &desc);
 
         float fConstData[][4] = {
-            {(float)desc.Width, (float)desc.Height, (float)(counter++), (float)diff / CLOCKS_PER_SEC},
-            {1.0f / desc.Width, 1.0f / desc.Height, 0, 0},
+            { (float)desc.Width, (float)desc.Height, (float)(counter++), (float)diff / CLOCKS_PER_SEC },

```

Comment:
That version doesn't exist? Personally, I'm still in 2.05.1, no idea if there's any newer, but you shouldn't do manual corrections in style since it will be overwritten the next time.